compliance/aws/untagged_resources/aws_untagged_resources.pt

```diff
@@ -38,8 +39,9 @@ end
 
 parameter "param_tags_to_check" do
   type "list"
-  label "Tags"
-  description "List of tags of format 'Key=Value' against which the resources will be compared."
+  label "Tag Keys"
+  category "AWS"
+  description "List of tag keys e.g., 'Environment' to find resources which are not tagged by the given inputs."
```

Comment:
Add `constraint_description` to express this as a mandatory field, to remain consistent with Azure counterpart

Reply:
Resolved